extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -337,18 +346,22 @@ export const AzureProvisionDialog: React.FC = () => {
     setPage(page);
     switch (page) {
       case PageTypes.AddResources:
+        telemetryClient.track('ProvisionAddResources', {});
```

Comment:
Maybe ProvisionAddResourcesPage or ProvisionAddResourcesNavigate to distinguish between an event of actually adding resources?